### ISSUE TITLE
Unexpected identifier 'enableVueLoader'

### ISSUE THREAD

**Issue by KriBetko** [symfony/webpack-encore#74]
I start the Encore `./node_modules/.bin/encore dev` and get this error:

> /home/kribeetko/PhpstormProjects/ESUNZ/node_modules/@symfony/webpack-encore/index.js:378
>     enableVueLoader(vueLoaderOptionsCallback = () => {}) {
>     ^^^^^^^^^^^^^^^
> SyntaxError: Unexpected identifier
>     at createScript (vm.js:56:10)
>     at Object.runInThisContext (vm.js:97:10)
>     at Module._compile (module.js:542:28)
>     at Object.Module._extensions..js (module.js:579:10)
>     at Module.load (module.js:487:32)
>     at tryModuleLoad (module.js:446:12)
>     at Function.Module._load (module.js:438:3)
>     at Module.require (module.js:497:17)
>     at require (internal/module.js:20:19)
>     at Object.<anonymous> (/home/kribeetko/PhpstormProjects/ESUNZ/webpack.config.js:1:76)

My `webpack.config.js`:
```
let Encore = require("@symfony/webpack-encore");

Encore
    .setOutputPath('web/build/')
    .setPublicPath('/build')
    .cleanupOutputBeforeBuild()
    .addEntry('scripts', './assets/app.js')
    .addStyleEntry('styles', './assets/app.scss')
    .enableVersioning()
    .enableSassLoader({
        resolve_url_loader: false
    })
    .enablePostCssLoader()
    .autoProvidejQuery()
    .autoProvideVariables({
        $: 'jquery',
        jQuery: 'jquery',
        'window.jQuery': 'jquery'
    })
    .configureBabel(function(babelConfig) {
        babelConfig.presets.push('es2017');
    })
    .enableSourceMaps(!Encore.isProduction());

module.exports = Encore.getWebpackConfig();
```

**Comment by hktr92** [comment]
I encountered the issue, too. It's in the latest version of Encore (0.9.0). [Symfony Demo](https://github.com/symfony/symfony-demo) uses 0.7.2 and it works without any issues.

![screenshot_20170701_234253](https://user-images.githubusercontent.com/7689151/27765390-5250b1e2-5eb8-11e7-8e25-3517a7c794fd.png)

**Comment by GZep** [comment]
[You can fix it by yourself for now.](https://github.com/symfony/webpack-encore/pull/70/files)

It's already fifth pull requests are exists for this issue.

**Comment by weaverryan** [comment]
Sorry about that! Fixed in 0.9.1 (run `yarn upgrade`) and we updated CI to catch this in the future :).

Cheers!
